packages/cli/src/util/runner.ts

```diff
@@ -0,0 +1,38 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { IPrismHttpServer } from '@stoplight/prism-http-server/src/types';
+import * as chokidar from 'chokidar';
+import { CreatePrismOptions } from './createServer';
+
+export function runPrism(createPrism: Function, options: CreatePrismOptions, spec: string) {
+  return createPrism(options).then((server: IPrismHttpServer) => {
+    if (server) {
+      const watcher = chokidar.watch(spec, { awaitWriteFinish: { stabilityThreshold: 500, pollInterval: 100 } });
```

Comment:
`awaitWriteFinish` was needed because sometimes `fs.readFileSync(file, { encoding: 'utf8' })` (https://github.com/stoplightio/prism/blob/dbe60ed1a1b7cd62a6778dbe70d18913c6f8e14d/packages/http/src/getHttpOperations.ts#L24) would give back an empty string, so, most probably the file was being accessed in the middle of it being saved